acceptance/analyzer_test.go

```diff
@@ -1122,7 +1122,7 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 
 			it("uses the group path at the layers path and writes analyzed.toml at the layers path", func() {
 				h.SkipIf(t,
-					api.MustParse(platformAPI).Compare(api.MustParse("0.5")) != 0 && api.MustParse(platformAPI).Compare(api.MustParse("0.6")) != 0,
+					api.MustParse(platformAPI).LessThan("0.5") || api.MustParse(platformAPI).AtLeast("0.7"),
```

Comment:
I guess there is no real chance that the following scenario will happen but I'm documenting it just in case:
If we publish new platform APIs like "0.5.x" or "0.6.x", we might need to fix this condition.

Reply:
But why? `Compare` takes into account the minor versions: https://github.com/buildpacks/lifecycle/blob/9895acb6a6169013c8eb5f58986e2f49ae478446/api/version.go#L90. The patch version shouldn't matter.

Reply:
You're right. I was a bit confused with the `Compare` function.